tests/cross_functional/system_test/test_full_cluster_health.py

```
@@ -4,108 +4,108 @@
+        if not sample.wait_for_func_status(result=True):
+            logger.error(
+                f"Cluster capacity did not reach 85% after {self.TIMEOUT_BENCHMARK_SETUP}s timeout"
+            )
+            raise TimeoutExpiredError
 
-        try:
-            if "Fio failed to execute" not in self.test_logs:
-                logger.info("FIO has completed successfully")
-        except IOError:
-            logger.warning("FIO failed to complete")
+        def teardown():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `teardown`

Add return type annotation: `None`

(ANN202)